### ISSUE TITLE
group of contigs [sf#125]

### ISSUE THREAD

**Issue by srynobio** [The-Sequence-Ontology/SO-Ontologies#125]
**Reported by alpapan on 2008-09-30 14:31 UTC**
Hello

I'm trying to assign a group of contigs \(assembled sequences\) as potentially belonging together in a higher but /unassembled/ group \(so obviously not supercontig but perhaps a similar hierarchy? 

I was wondering what would be the best way to represent this?

I was thinking of sequence\_collection &gt; contig\_group &gt; contig ?

many thanks\!
a

**Comment by srynobio** [comment]
**Commented by eilbeck on 2008-12-16 00:15 UTC**
Hi Alexie,
Sorry for not getting to you sooner, but I went on maternity leave the day before you posted this and it went to the wrong tracker, so I just noticed it now.
I think what you want is a group of non contiguous contigs.

Let me get back to you. I'm not sure if the rest of the community is entirely happy with the sequence\_collection term.

\--Karen
